dist/index.html

```diff
@@ -0,0 +1,18 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>To Do List</title>
+<script defer src="runtime.js"></script><script defer src="main.js"></script></head>
+<body>
+
+  
+    <ul class="todo-list">
+      <li>Today's To Do</li>
+      <li>
+        <input type="text" name="" id="add-to-list" placeholder="Add to your list..." id="">
+      </li>
+    </ul>
```

Comment:
Template
![image](https://user-images.githubusercontent.com/78251927/236040064-8a4c3235-9b21-4c20-a674-a297e0139554.png)

Your design
![image](https://user-images.githubusercontent.com/78251927/236040364-9057a880-5875-48bc-9331-98d14f729df9.png)


According to the project requirement, it is important that our design is similar to the list using the [captured video](https://www.youtube.com/watch?v=AcUd-_Yjjqg). 

![image](https://user-images.githubusercontent.com/78251927/235740491-7fe3ce92-bb04-4085-8055-a10bb13c9e1a.png)

Please kindly add the missing icon in your design 💪